metagraph/src/graph/representation/canonical_dbg.cpp

```diff
@@ -228,7 +237,7 @@ ::call_incoming_kmers(node_index node, const IncomingEdgeCallback &callback) con
                 ::reverse_complement(&c, &c + 1);
 
                 rev_seq.back() = c;
-                prev = map_sequence_to_nodes(graph_, rev_seq)[0];
+                prev = graph_.kmer_to_node(rev_seq);
```

Comment:
Why did it call `map_sequence_to_nodes` before?

Did we need to map it to canonical k-mers, so `kmer_to_node` did not work?

Reply:
I think it was just an oversight on my part. `map_sequence_to_nodes` doesn't map to canonical k-mers anyway, so these two are functionally equivalent.